testing/tests/DevExpress.ui.widgets.editors/textEditorParts/mask.tests.js

```diff
@@ -1153,6 +1153,33 @@ QUnit.module("value", moduleConfig, () => {
         assert.equal(valueChangedFired, 1, "change fired once on pressing enter key");
     });
 
+    QUnit.test("value change by option should reset _changedValue (T833415)", (assert) => {
```

Comment:
Let's change the test subject to reflects the use case instead of the internal implementation.